doc/sphinx/addendum/type-classes.rst

```diff
@@ -223,23 +221,21 @@ this declaration is equivalent to:
       { le : A -> A -> bool }.
 
 
-This declaration means that any instance of the ``Ord`` class must have
-an instance of ``EqDec``. The parameters of the subclass contain at
-least all the parameters of its superclasses in their order of
-appearance (here A is the only one). As we have seen, ``Ord`` is encoded
-as a record type with two parameters: a type ``A`` and an ``E`` of type
-``EqDec A``. However, one can still use it as if it had a single
-parameter inside generalizing binders: the generalization of
+This declaration means that any instance of the ``Ord`` typeclass must have an
+instance of ``EqDec``. The parameters of the subclass contain at least all the
+parameters of its superclasses in their order of appearance (here A is the only
+one). As we have seen, ``Ord`` is encoded as a record type with two parameters:
+a type ``A`` and an ``E`` of type ``EqDec A``. However, one can still use it as
+if it had a single parameter inside generalizing binders: the generalization of
```

Comment:
Would be better only to change the first line here and not reformat the rest of the paragraph.  That way I don't have to mentally diff the old and the new to find what's been changed.

Reply:
Yes that is true, but I also really don't like having inconsistent wrapping. If I don't change it now, who will change it?

Reply:
One upside is that when we edit this doc in the future you won't have to complain about the wrapping changes.

Reply:
Can you reverse the wrapping?  It obscures the git history.  I don't rewrap lines I haven't changed.  Same thing for reformatting source code--not recommended.